src/main/kotlin/lotto/Application.kt

```diff
@@ -0,0 +1,24 @@
+package lotto
+
+import view.InputView
+import view.ResultView
+
+fun main() {
+    val amountInput = InputView.getAmountInput()
+    val purchaseAmount = Money(amountInput)
+    val lottoMachine = LottoMachine(purchaseAmount)
+    val lottoTickets = lottoMachine.tickets
+
+    ResultView.viewTicketsAmount(lottoTickets.size)
+    ResultView.viewTickets(lottoTickets)
+
+    val winningNumbers = Lotto(InputView.getWinningNumbers())
+    val bonusNumber = InputView.getBonusNumber()
+
+    val game = LottoGame(lottoTickets, winningNumbers, bonusNumber)
```

Comment:
I think the `Lotto`'s six numbers and bonusNumber are the same thing.
so bonusNumber need validation too.

How about wrapping both of them with the same VO (Value Object) and moving the validation there?
like the `Money` class!